pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -410,6 +422,8 @@ impl RemoteTimelineClient {
         )
         .await?;
 
+        drop(unfinished_gauge_guard);
```

Comment:
instead of dropping, can we use code block, i.e :  {
{
  let unfinished_gauge_guard = self.metrics.call_begin(&RemoteOpFileKind::Layer, &RemoteOpKind::Download);

   let downloaded_size = download::download_layer_file(
            self.conf,
            &self.storage_impl,
	@@ -410,6 +422,8 @@ impl RemoteTimelineClient {
        )
        .await?;

} // not needed anymore as goes out of scope :        drop(unfinished_gauge_guard);